srtcore/epoll.cpp

```diff
@@ -478,6 +495,47 @@ int CEPoll::wait(const int eid, set<UDTSOCKET>* readfds, set<UDTSOCKET>* writefd
                ++ total;
             }
          }
+         #elif defined(WIN32)
+
+         DWORD ready;
+         DWORD timeout_ms = 10;
+         const int nhandles = p->second.m_mLocals.size();
+         HANDLE handles[MAXIMUM_WAIT_OBJECTS];
+         int i = 0;
+
+         for (std::map<SYSSOCKET, CEPollFD>::iterator it = p->second.m_mLocals.begin();
+              it != p->second.m_mLocals.end();
+              ++it)
+         {
+            handles[i++] = (HANDLE) it->first;
+         }
+
+         ready = WaitForMultipleObjectsEx(nhandles, handles, FALSE, timeout_ms, TRUE);
```

Comment:
I am worried this changes the meaning of SYSSOCKET: it will no longer be an I/O object but an event object that is indeed missing sense of direction. In my understanding, I can't pass handle created with socket() to WaitForMultipleObjectsEx()? And if so, this is no longer a 'SYSSOCKET' but 'SYSEVENT' so API meaning and usage has changed?